### PR TITLE
Setup minimal python package with setup.py

### PR DESCRIPTION
**Pull request by atemate**
Turn code into a simple Python package with `setup.py`:
- move all code to `./pasd`, fix imports (for example turn `from pipelines.pipeline_pasd import...` to `from pasd.pipelines.pipeline_pasd import...`)
- add `setup.py` and `requirements.txt`
- move dependency `salesforce-lavis` to a separate file `requirements-test.txt` as it brings conflict:
```
The conflict is caused by:
    salesforce-lavis 1.0.2 depends on decord
    salesforce-lavis 1.0.0 depends on decord
```

See installation instructions in `README.md`

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,4 +8,3 @@ open_clip_torch
 einops
 gradio<4.0.0
```
Comment (atemate) [inline requirements.txt:9]: Also note, some files like `net.py` import `torchvision` which is not listed in `requirements.txt`. Is it a bug?

File: requirements.txt [hunk 2]
```diff
@@ -1,11 +1,10 @@
 diffusers==0.29.2
 accelerate
 transformers
-xformers
+xformers; sys_platform != 'darwin'
```
Comment (atemate) [inline requirements.txt:4]: xformers can't be installed on Mac: https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/11396

As far as I understand, this dependency is not imported explicitly in the code and the model can do inference on cpu on Mac, just very slow

**Comment by atemate** [comment]
Hey @Atry @yangxy would you please consider reviewing and merging this PR? It enables `pip` to be able to install this repo as a Python package, which is critical for package managers like `poetry`

**Comment by yangxy** [comment]
> Hey @Atry @yangxy would you please consider reviewing and merging this PR? It enables `pip` to be able to install this repo as a Python package, which is critical for package managers like `poetry`

could you please fix the conflicts with the latest repo?

**Comment by atemate** [comment]
hey @yangxy thank you for noticing my PR! branch updated 👍

**Comment by atemate** [comment]
hi @yangxy @Atry let's merge this PR?

**Comment by yangxy** [comment]
> hi @yangxy @Atry let's merge this PR?

done. sorry for the delay.

**Pull request merged**